[FIX] Markdown message preview

LucasSiqz opened the pull request:
<!-- INSTRUCTION: Keep the line below to notify all core developers about this new PR -->
@RocketChat/ReactNative

<!-- INSTRUCTION: Inform the issue number that this PR closes, or remove the line below -->

<!-- INSTRUCTION: Tell us more about your PR with screen shots if you can -->
Before changes:

![image](https://user-images.githubusercontent.com/23382026/67775073-290a4a00-fa3d-11e9-825f-811234c7a8a7.png)


![image](https://user-images.githubusercontent.com/23382026/67774939-00825000-fa3d-11e9-92ff-256295a923cb.png)


Image preview wrong and preview markdown messages with two white spaces


After changes: 

![image](https://user-images.githubusercontent.com/23382026/67774988-0e37d580-fa3d-11e9-97db-05772bf78cd1.png)

![image](https://user-images.githubusercontent.com/23382026/67775009-14c64d00-fa3d-11e9-9fcd-be3fcebe911f.png)

diegolmello commented on app/containers/markdown/index.js:
```diff
@@ -151,7 +151,7 @@ export default class Markdown extends PureComponent {
 			</Text>
 		);
-	}
+	};
```
Can you remove these semicolons?

LucasSiqz replied:
yes, it was my prettier extension, sorry xD

diegolmello commented on app/containers/markdown/index.js:
```diff
@@ -317,7 +324,7 @@ export default class Markdown extends PureComponent {
 		// Ex: '[ ](https://open.rocket.chat/group/test?msg=abcdef)  Test'
 		// Return: 'Test'
-		m = m.replace(/^\[([\s]]*)\]\(([^)]*)\)\s/, '').trim();
+		m = m.replace(/(\s)\[([\s]]*)\]\(([^)]*)\)\s/, '').trim();
```
What are you doing here?

LucasSiqz replied:
I saw that \s covers spaces and added it to regex and apparently worked for this case, but seeing better now, it is failing in the example above. Do you have any suggestion on how to solve this problem?

diegolmello commented on app/presentation/RoomItem/LastMessage.js:
```diff
@@ -35,7 +35,7 @@ const formatMsg = ({
 	}
 
-	let msg = `${ prefix }${ lastMessage.msg.replace(/[\n\t\r]/igm, '') }`;
+	let msg = `${ prefix }${ lastMessage.msg.replace(/[\n\t\r]/igm, '').trim() }`;
```
What are you doing here?

LucasSiqz replied:
it was just a unsuccessfully tried to remove the two white spaces in markdown message preview that i forgot to remove.

diegolmello commented:
@LucasSiqz I removed your images from description because of inappropriate language. Feel free to add another. Thanks.

LucasSiqz commented:
> @LucasSiqz I removed your images from description because of inappropriate language. Feel free to add another. Thanks.

I updated the images, sorry :/
